### ISSUE TITLE
Context wierd when editing long file

### ISSUE THREAD

**Issue by tkkcc** [nvim-treesitter/nvim-treesitter-context#97]
Don't know if it's caused by `treesitter` or `context`. Have to disable `context` now.

nvim version:
```txt
NVIM v0.6.1
Build type: Release
LuaJIT 2.1.0-beta3
Compiled by builduser

Features: +acl +iconv +tui
See ":help feature-compile"

   system vimrc file: "$VIM/sysinit.vim"
  fall-back for $VIM: "/usr/share/nvim"

Run :checkhealth for more info
```

minimal init.lua to reproduce:
```lua
require('packer').startup(function()
  use 'wbthomason/packer.nvim'
  use {
    'nvim-treesitter/nvim-treesitter',
    run = ':TSUpdate',
    config = function()
      require'nvim-treesitter.configs'.setup {
        ensure_installed = "maintained",
      }
    end,
  }
  use {
    'romgrk/nvim-treesitter-context',
    config = function()
      require'treesitter-context.config'.setup {enable = true}
    end,
  }
end)

```

file:
https://raw.githubusercontent.com/tkkcc/arknights/c154c07dcccda63140523d13d908d22ba8c0de6f/path.lua

result:
![Peek 2022-02-11 15-21](https://user-images.githubusercontent.com/17373509/153551824-5d47e284-a241-43ee-9b1a-9bf7b62b69fc.gif)

**Comment by binhtran432k** [comment]
I don't know why you use Unicode in variable name, etc. I think treesitter not recognize it as the variable and concat with below line which make context line is wierd.

**Comment by binhtran432k** [comment]
I don't know exactly but I just rename the identifier on line 13 then the context work correctly. But after few line it become weird again because later have similar problem.

**Comment by lewis6991** [comment]
Should be fixed in #198
